packages/opencode/src/tool/read.txt

```diff
@@ -6,7 +6,7 @@ Usage:
 - By default, it reads up to 2000 lines starting from the beginning of the file
 - You can optionally specify a line offset and limit (especially handy for long files), but it's recommended to read the whole file by not providing these parameters
 - Any lines longer than 2000 characters will be truncated
-- Results are returned using cat -n format, with line numbers starting at 1
+- Results are returned using cat -n format, with line numbers starting at 1. When using this output with other tools (e.g., `edit`), you MUST strip these line number prefixes and the subsequent tab character.
```

Comment:
> and the subsequent tab character

I wonder if this could cause some sort of regression for other models. (i.e. removing indents too aggressively) 🤔